cordova-lib/src/plugman/fetch.js

```diff
@@ -257,7 +237,7 @@ function checkID(expectedIdAndVersion, pinfo) {
     var parsedSpec = pluginSpec.parse(expectedIdAndVersion);
 
     if (parsedSpec.id != pinfo.id) {
-        var alias = parsedSpec.scope ? null : pluginMappernto[parsedSpec.id] || pluginMapperotn[parsedSpec.id];
+        var alias = null;
         if (alias !== pinfo.id) {
```

Comment:
kk thanks! @stevengill  If you remove this code, there are tests that go with it and they will fail so I removed those in the next commit. How does this look? https://github.com/apache/cordova-lib/pull/522/commits/a29720e6d6b2378d9e342a5b1bb46bef1db17fa8